modules/oauth-api/src/main/java/org/mule/runtime/oauth/api/builder/OAuthPlatformManagedDancerBuilder.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.oauth.api.builder;
+
+import org.mule.api.annotation.Experimental;
+import org.mule.api.annotation.NoImplement;
+import org.mule.runtime.api.tls.TlsContextFactory;
+import org.mule.runtime.http.api.client.HttpClient;
+import org.mule.runtime.http.api.client.proxy.ProxyConfig;
+import org.mule.runtime.oauth.api.PlatformManagedOAuthDancer;
+import org.mule.runtime.oauth.api.listener.PlatformManagedOAuthStateListener;
+
+import java.nio.charset.Charset;
+import java.util.function.Function;
+
+/**
+ * Builder that allows to configure the attributes for the {@link PlatformManagedOAuthDancer}
+ * <p>
+ * Because the actual OAuth authorization is not performed by Mule but the Anypoint Platform, some of the methods in the base
+ * {@link OAuthDancerBuilder} do not apply and will throw an {@link UnsupportedOperationException}. The the javadocs of all
+ * methods defined and overridden in this interface to know which ones are valid.
+ * <p>
+ * Platform Managed OAuth is an experimental feature. It will only be enabled on selected environments and scenarios.
+ * Backwards compatibility is not guaranteed.
+ *
+ * @since 4.3.0
+ */
+@NoImplement
```

Comment:
won't this implementation be in the service artifact?

Reply:
on the EE one